openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/UsersTab/UsersTabs.component.tsx

```diff
@@ -139,7 +139,7 @@ export const UsersTab = ({ users, onRemoveUser }: UsersTabProps) => {
           ),
         }}
         pagination={false}
-        rowKey="fullyQualifiedName"
```

Comment:
Please check, incase we are using it fqn somewhere!

Reply:
Checked only used for Persona